frontend/pendingTranslations.ftl

```diff
@@ -152,7 +152,7 @@ whatsnew-feature-bundle-snippet = Upgrade your subscription to get both { -brand
 # Variables:
 #   $monthly_price (string) - the monthly cost (including currency symbol) for a given plan. Examples: $0.99, 0,99 €
 #   $savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
-whatsnew-feature-bundle-body = Upgrade your subscription to get both { -brand-name-firefox-relay-premium } + { -brand-name-mozilla-vpn } for { $monthly_price }/month. Save { $savings }!
+whatsnew-feature-bundle-body = Upgrade your subscription to get both { -brand-name-firefox-relay-premium } + { -brand-name-mozilla-vpn } for { $monthly_price }/month. { $savings } off Mozilla VPN!
```

Comment:
```suggestion
whatsnew-feature-bundle-body = Upgrade your subscription to get both { -brand-name-firefox-relay-premium } + { -brand-name-mozilla-vpn } for { $monthly_price }/month. { $savings } off { -brand-name-mozilla-vpn }!
```

That said, note that this string [has already been submitted to the l10n repository](https://github.com/mozilla-l10n/fx-private-relay-l10n/blob/c6d867777f4a4ddb8998bf4c1ccc62995408c25a/en/bundle.ftl#L19), so you'll need to add a new string (also for the other changed string).